src/services/upload.services.js

```diff
@@ -33,6 +33,39 @@ class UploadService {
 		return blobLink;
 	};
 
+	// FIXME
+	// if move url image from content to thumbnail that lead to in the future when 
```

Comment:
I think you should remove this fixme because in case convert from content to thumbnail it create a new record of thumbnail so that It still get list of images for the post's content